### PR TITLE
Rocket.chat file upload support

### PR DESCRIPTION
**Pull request by mfilotto**
This PR implements #69

### CODE REVIEW

File: src/grafana.coffee [hunk 1]
```diff
@@ -396,6 +403,55 @@ module.exports = (robot) ->
                 robot.logger.error "Slack service error while posting data:" +res["error"]
                 msg.send "#{title} - [Form Error: can't upload file] - #{link}"
 
+    'rocketchat': (msg, title, grafanaDashboardRequest, link) ->
```
Comment (stephenyeargin) [inline src/grafana.coffee]: This is more commentary than anything that needs to be changed in this PR, but some of these adapter-specific pieces (Slack, Rocketchat and perhaps even the S3 upload) should likely be divided out into their own classes/source files somehow. The package's line count is getting on up there.

Comment (mfilotto) [reply src/grafana.coffee]: Let me know what kind of refactoring you want

Comment (stephenyeargin) [reply src/grafana.coffee]: Let's call it out of scope for this one. I was looking at the way some of it was done in https://github.com/hubot-scripts/hubot-pager-me/tree/master/src , but it's probably not the best example.

**Comment by stephenyeargin** [comment]
Released with `v2.1.0`.

**Pull request merged**